tests/default/flow_framework/workflow.yaml

```
@@ -0,0 +1,59 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+description: Test flow_framework endpoints.
+version: '>=2.12'
```

```suggestion
version: '>= 2.12'
```

(I think we add a space, but check ;)